### DISCUSSION TITLE
Cannot save in console

### DISCUSSION THREAD

**Discussion by andredev93** [mia-platform/community#230] [Q&A] [answered]
### Which section?

Console

### Which is the version?

10.7

### Description

Hi,

I'm getting an error while saving on an On-prem installation of the console with Github integration.

In detail, after making some changes and saving I receive a 400 Bad request with the message: "Unable to save configuration".

![image](https://user-images.githubusercontent.com/10067183/231476383-d3a9b58f-85d2-40a0-af9f-9cba9aaa9894.png)

From backend logs we see these errors:

`"data":{"message":"Validation Failed","errors":[{"resource":"author","code":"missing_field","field":"name"}],"documentation_url":"https://docs.github.com/enterprise-server@3.7/rest/reference/git#create-a-commit"}}`

`\"author\":{\"name\":\"\",\"email\":\"<redacted>\"}}","request":{}}},..."message":"aaa","msg":"unable to create a commit on github"}`

### Actual Outcome

Error while saving on the console.

### Expected Outcome

Changes saved successfully.

**Comment by fredmaggiowski** [comment]
HI this likely seems like an issue with your authenticated user or an issue with your GitHub provider configuration. Since the error is saying that the author name for the commit it looks like your user has no name in the users collection, can you verify that?

**Reply by andredev93** [reply, marked as answer]
Hi, it looks like it was missing the email in the profile in Github. Once added and refreshed the console page, the save feature started working normally.
Thanks.
